### ISSUE TITLE
Error when creating the Trainer

### ISSUE THREAD

**Issue by MaxEAB** [Deci-AI/super-gradients#1055]
### 💡 Your Question

Hi, I am trying to train pp_lite_t_seg50 on my own custom data. The Trainer is created as,

```
from super_gradients import setup_device
from super_gradients.training import Trainer, MultiGPUMode

setup_device(multi_gpu=MultiGPUMode.OFF, num_gpus=1)
trainer = Trainer(experiment_name='cust_ppLite')
```
This results in the following error:

"Exception has occurred: TypeError
Accuracy.__new__() missing 1 required positional argument: 'task'"

Is this due to using an incompatible Torch version? I am on torch 2.0.1+cu117.

**Comment by Louis-Dupont** [comment]
Hi @MaxEAB , this works all fine on my device

```
Python 3.10.4

torch                         2.0.1
torchmetrics                  0.8.0
torchvision                   0.15.2
```

My quick guess is that your python version might not be compatible with `torch 2.0.1`, but it might be something else.
Can you please share your python, torchmetrics and torchvision versions ? Also if you can show the whole traceback, it would help me see see what triggers it.

**Comment by MaxEAB** [comment]
Switching to torch 1.13.1 fixed this.
